src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -185,7 +165,6 @@ class EthService(
 
   import EthService._
 
-  val hashRate: AtomicReference[Map[ByteString, (BigInt, Date)]] = new AtomicReference[Map[ByteString, (BigInt, Date)]](Map())
   val lastActive = new AtomicReference[Option[Date]](None)
```

Comment:
This is only being used by a now unused function `reportActive` , should we maybe delete both of them?

Reply:
:+1: